nextcord/message.py

```diff
@@ -1237,13 +1237,19 @@ async def edit(
             the view is removed.
         file: Optional[:class:`File`]
             A new file to add to the message.
+
+            .. versionadded:: 2.0
         files: Optional[List[:class:`File`]]
             A list of new files to add to the message.
+
+            .. versionadded:: 2.0
         append_files: Optional[:class:`bool`]
```

Comment:
Include default value?
lgtm, tested!

Reply:
There is no default value though, updated the docs a little bit though, does that seem correct?

Reply:
Just wondering what happens if I don't pass in `append_files`
Defaults to True, hence including that would be a good idea.

Reply:
Default for `append_files` is already there??

Reply:
"If set to True (default)"